frontend/src/main/pages/AdminUsersPage.js

```diff
@@ -6,11 +6,12 @@ import { useUsers } from "main/utils/users";
 const AdminUsersPage = () => {
 
     const { data: users } = useUsers();
```

Comment:
```suggestion
 const { data: users, error: _error, status: _status } =
        useBackend(
            // Stryker disable next-line all : don't test internal caching of React Query
            ["/api/admin/users"],
            // Stryker disable next-line StringLiteral,ObjectLiteral : since "GET" is default, "" is an equivalent mutation
            { method: "GET", url: "/api/admin/users" },
            []
        );

```

Reply:
After you do this, you can get rid of the file from which useUsers is imported, as well as any tests associated with that file.   We don't need it; and having it around actually makes the code base worse.  The useBackend hook handles the caching correctly.   The bug in your page is because the code isn't using it (for reasons that are not clear.)